source/linear-algebra/source/01-LE/03.ptx

```diff
@@ -234,16 +255,7 @@ How many solutions must this system have?
 <sage language="octave">
 </sage>
 
-<fact xml:id="linearsystemsfact2">
-    <statement>
-        <p>
-We will see in <xref ref="LE4"/> that the intuition established here generalizes: a <em>consistent</em> system with more variables
-than equations (ignoring <m>0=0</m>) will always have infinitely many solutions.
-        </p>
-    </statement>
-</fact>
```

Comment:
This felt a bit duplicative with the following Fact. But @jkostiuk should chime in - it's not clear to me what you're asking for in your Additional Practice activity that cites this (which I updated to cite the next fact).

Reply:
ya, i don't disagree and I like how the fact is stated now a bit better. I think I might have to think of a better individual practice for the future.